ambuda/static/js/core.ts

```diff
@@ -27,14 +26,16 @@ function forEachTextNode(elem, callback) {
     const node = nodeList[i];
     if (node.nodeType === Node.TEXT_NODE) {
       node.textContent = callback(node.textContent);
-    } else if (node.lang !== 'en') {
+    } else if (!node.lang || node.lang === 'sa') {
```

Comment:
Why this change? What does `!node.lang` mean?

Reply:
Updated the function comments to clarify.